src/room.ts

```diff
@@ -356,31 +396,60 @@ export class Room extends EventEmitter implements Sayable {
   }
 
   /**
-   * NickName / DisplayName / RemarkName of member
+   * find member by `nick`(NickName) / `display`(DisplayName) / `remark`(RemarkName)
+   * when use member(name:string), find member by nickName by default
    */
-  public member(name: string): Contact | null {
-    log.verbose('Room', 'member(%s)', name)
+  public member(queryArg: MemberQueryFilter | string): Contact | null {
+    if (typeof queryArg === 'string') {
+      log.error('Room', 'function member should use member(queryArg: MemberQueryFilter)')
```

Comment:
Should not use `log.error()` here because it is not a bug/error, it's a feature.